src/main/java/net/pms/network/Request.java

```diff
@@ -327,10 +329,16 @@ public void answer(
 							// XXX external file is null if the first subtitle track is embedded:
 							// http://www.ps3mediaserver.org/forum/viewtopic.php?f=3&t=15805&p=75534#p75534
 							if (sub.isExternal()) {
-								inputStream = new FileInputStream(sub.getExternalFile());
+								if (sub.getType() == SubtitleType.SUBRIP && mediaRenderer.isRemoveTagsFromSRTsubs()) { // remove tags from .srt subs when renderer doesn't support them
+									inputStream = new FileInputStream(SubtitleUtils.removeTagsFromSubs(sub.getExternalFile()));
+								} else {
+									inputStream = new FileInputStream(sub.getExternalFile());
+								}
+							} else {
+								LOGGER.trace("Not loading external subtitles because they are not external: " + sub);
 							}
-						} catch (NullPointerException npe) {
-							LOGGER.trace("Could not find external subtitles: " + sub);
+						} catch (IOException npe) {
```

Comment:
@Nadahar I will add it but I can't imagine which part of the code can throws NPE